css/animations_transitions.css

```diff
@@ -46,6 +46,11 @@
   background: #008552;
 }
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/231651257-814bc150-ba74-44d7-b157-06ccbf5f3a75.png)

- [ ] It would be nice if you make your cards align in the center of the page and reduce some more space from the right side :+1: as suggested by the previous reviewer to make your project follow the Figma template to make your project professional :blush:
- [ ] Kindly make your title **(Pokedex Personal Work)** color as you can see in the Figma template as suggested by the previous reviewer :+1: